internal/pkg/docker/dockerfile/dockerfile.go

```diff
@@ -169,3 +216,34 @@ func parseExpose(line string) []portConfig {
 	}
 	return ports
 }
+
+func parseHealthCheck(line string) (healthCheck, error) {
```

Comment:
I think perhaps it would be better to keep this method's signature the same as `parseExpose` for consistency. `parseExpose` includes error in `portConfig` struct.